scripts/linters/pylint_extensions.py

```diff
@@ -44,7 +44,7 @@
 # comment or docstring.
 EXCLUDED_PHRASES = [
     'coding:', 'pylint:', 'http://', 'https://', 'scripts/', 'extract_node',
-    'type:'
+    'type:', 'pragma:', 'isort:'
```

Comment:
I believe we would have pragmas for `type:`, `pragma:`, `isort:` and `pylint:` only and not for the rest. Can you make a new list instead and use that to check whether to allow trailing comments to start with these or not?